backend/Controllers/ManagerDashboardController.cs

```diff
@@ -112,4 +112,23 @@ public IActionResult GetEmployeeListAll()
 
         return Ok(queryResult);
     }
+
+    //GET request to get specific status of employee
+    [HttpGet]
+    [Route("GetEmployeeStatus")]
+    public IActionResult GetEmployeeStatus(int employee_HID)
+    {
+        var review = default(Review);
+        //find the current review for the employee
+        review = _dbContext.Reviews.FirstOrDefault(review => review.EmployeeHID == employee_HID);
```

Comment:
this will be something like review = _dbContext.Reviews.FirstOrDefault(review => review.EmployeeHID == employee_HID && review.Status != "Finalized"), this is because we want to find the current review (all finished reviews have "Finalized" status so there will only be one that is current for the specific employee). 

Otherwise, with your current query, you will get a list back of all their reviews including current and past and randomly select the first one which won't give you the result you want.

Reply:
that makes sense ill do that 😎